docs/using_starrocks/External_table.md

```diff
@@ -232,11 +232,11 @@ select * from es_table where esquery(k4, ' {
 
 ## External table for a JDBC-compatible database
 
-From v2.3.0, StarRocks provides external tables to query JDBC-compatible databases. This way, you can analyze the data of such databases in a blazing fast manner without the need to import the data into StarRocks.  This topic describes how to create an external table in StarRocks and query data in JDBC-compatible databases.
+From v2.3.0, StarRocks provides external tables to query JDBC-compatible databases. This way, you can analyze the data of such databases in a blazing fast manner without the need to import the data into StarRocks. This topic describes how to create an external table in StarRocks and query data in JDBC-compatible databases.
 
 ### Prerequisites
 
-When a JDBC resource is created, and a JDBC external table is queried for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are located must have access to the download URL of the JDBC driver during these two phases. The download URL is specified by the `driver_url` parameter when the JDBC resource is created.
+When a JDBC resource is created, and a JDBC external table is queried for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are located must have access to the download URL of the JDBC driver JAR package during these two phases. The download URL is specified by the `driver_url` parameter when the JDBC resource is created.
```

Comment:
When you create a JDBC resource or use a JDBC external table to query data for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are deployed must have access to the download URL of the JDBC driver. The download URL is specified by the `driver_url` parameter in the statement used for creating the JDBC resource.